addons/packages/pinniped/0.12.1/bundle/config/values.yaml

```diff
@@ -15,6 +15,14 @@ imageInfo:
       imagePath: dex
       tag: v2.27.0_vmware.1
 infrastructure_provider: null
+nodeSelector: null
```

Comment:
Given that github issue, should we have a "why" comment indicating the reasoning for the addition?

Reply:
The comments are added in the overlay file

Reply:
great!